RELEASE.md

```diff
@@ -6,6 +6,10 @@ Please follow the established format:
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
 
+## Bug fixes and other changes
+
+- Fix lambda Python functions not rendering correctly on flowchart. (#851)
```

Comment:
```suggestion
- Fix lambda and partial Python functions not rendering correctly on flowchart. (#851)
```
The same fix here will work for both cases :)

Reply:
Nice one. Updated!